lib/access.js

```diff
@@ -34,152 +20,200 @@ const subcommands = [
   '2fa-not-required',
 ]
 
-const UsageError = (msg) =>
-  Object.assign(new Error(`\nUsage: ${msg}\n\n` + usage), {
-    code: 'EUSAGE',
-  })
-
-const cmd = (args, cb) =>
-  access(args)
-    .then(x => cb(null, x))
-    .catch(err => err.code === 'EUSAGE'
-      ? cb(err.message)
-      : cb(err)
+class Access {
+  constructor (npm) {
+    this.npm = npm
+  }
+
+  // TODO we don't use this, why are we exporting it?
```

Comment:
I don't believe we ever used it for `access`, but some other command(s) did once upon a time, iirc?  Config, I think?

Anyway, let's not export anything we're not using.  I say ditch it.